src/index.js

```diff
@@ -1,35 +1,93 @@
 import './style.css';
 import '@fortawesome/free-solid-svg-icons';
 
-const todoList = [
-  {
-    description: 'An example described in the string',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'An example described in the string',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'An example described in the string',
-    completed: true,
-    index: 2,
-  },
-];
-
-const displayTodos = (todo) => {
-  const ul = document.querySelector('ul');
-  let content = '';
-  todo.forEach((tod) => {
-    content += `<li class="item">
-        <input type="checkbox" />
-        <p contenteditable="true">${tod.description}</p>
-      </li>`;
+class Todo {
+  constructor(desc, completed, id) {
+    this.desc = desc;
+    this.completed = completed;
+    this.id = id;
+  }
+}
+class UI {
+  static displayTodos = (todos) => {
+    const ul = document.querySelector('ul');
+    if (!todos.length) ul.innerHTML = '';
+    let content = '';
+    todos.forEach((tod) => {
+      content += `
+        <li id=${tod.id} class="item">
+          <div class='group'>
+            <input class='bdan' name='checkbox' type="checkbox" />
+            <p contenteditable="true">${tod.desc}</p>
+          </div>
+          <i class="fa-solid fa-trash"></i>
+        </li>`;
+    });
     ul.innerHTML = content;
-  });
+    content = '';
+  };
 
-  content = '';
-};
-displayTodos(todoList);
+  static generateId = () => Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+
+  static clearValue = () => {
+    document.querySelector('.input').value = '';
+  };
+}
+
+class Store {
+  static getTodos = () => {
+    let todos;
+    if (!localStorage.getItem('todos')) todos = [];
+    else todos = JSON.parse(localStorage.getItem('todos'));
+    return todos;
+  };
+
+  static addTodo = (todo) => {
+    const todos = Store.getTodos();
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+    UI.displayTodos(todos);
+  };
```

Comment:
[OPTIONAL]
- [ ] You may also consider adding the list in such a way that when the user hit the `Enter` keyboard the item is added to the list :smile: 
`HINT:`
```
// Execute a function when the user presses a key on the keyboard
input.addEventListener("keypress", function(event) {
  // If the user presses the "Enter" key on the keyboard
  if (event.key === "Enter") {
    // Cancel the default action, if needed
    event.preventDefault();
    // Trigger the button element with a click
    document.getElementById("myBtn").click();
  }
});
```

Reply:
@christianbiring1  Thank you for your review. I have added this comment code cuz I'm going to use it on today's task